### ISSUE TITLE
Move relevant types from std/java to std/jvm and delete the rest

### ISSUE THREAD

**Issue by Simn** [HaxeFoundation/haxe#11565]
Reminder issue that we still need to do this.

**Comment by kLabz** [comment]
Did you want to keep them around for a while as proxies with `@:deprecated`?

**Comment by Simn** [comment]
No, if anything those could go into hx4compat. I don't know how many types there are that work on the JVM target, IIRC quite a few of them have `__java__` stuff.
